site/docs/master/restore-reference.md

```diff
@@ -9,6 +9,29 @@ velero restore create RESTORE_NAME \
   --from-backup BACKUP_NAME \
   --namespace-mappings old-ns-1:new-ns-1,old-ns-2:new-ns-2
 ```
+## What happens when user removes restore Objects. 
+A **restore** object represents the restore operation.There are two types of deletion for restore objects:
+### 1. Deleting with **`velero restore delete`**
+This command will delete the custom resource representing it, along with its individual log and results files.But, it will not delete any objects that were created by it from your cluster.
```

Comment:
"representing the restore, ...". Also please add a space after the period ("...results files. But, ...")